app/services/idv/acuant/assure_id.rb

```diff
@@ -25,6 +25,7 @@ def create_document
         )
 
         status, @instance_id = post(url, options) { |body| body.delete('"') }
+        @instance_id = Nokogiri::HTML(@instance_id).text
```

Comment:
can we add specs?

- I also confirmed via rails console that if the response ever goes back to just being the string (no HTML tags), that it will still behave as expected

Reply:
I added some tests in #3628 3628